api/_exclusivity/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { ethers } from "ethers";
+
+// TODO: get this from gh
+export const whiteListedRelayers = [
+  "0x9A8f92a830A5cB89a3816e3D267CB7791c16b04D", // dev wallet
+];
+
+export const getRelayerFromSignature = async (
```

Comment:
`verifyMessage()` isn't async; can we drop the `async` here ?

```suggestion
export const getRelayerFromSignature = (
```